src/ai/backend/agent/docker/agent.py

```diff
@@ -1685,6 +1687,37 @@ async def pull_image(
             elif error := result[-1].get("error"):
                 raise RuntimeError(f"Failed to pull image: {error}")
 
+    async def _purge_image(self, docker: Docker, image: str) -> PurgeImageResult:
+        try:
+            # QUESTION: Do we need to expose force, noprune option?
+            response = await docker.images.delete(image)
```

Comment:
What do you think about this part?
Do we need to expose force, noprune option? @HyeockJinKim

Reply:
Decided to proceed with the next task.